app/javascript/dashboard/store/modules/conversations/getters.js

```diff
@@ -9,10 +9,7 @@ export const getSelectedChatConversation = ({
 
 // getters
 const getters = {
-  getAllConversations: ({ allConversations }) =>
-    allConversations.sort(
-      (a, b) => b.messages.last()?.created_at - a.messages.last()?.created_at
-    ),
```

Comment:
I have removed this condition in favour of the sorting filter.
@pranavrajs @muhsin-k